hedera-node/hedera-mono-service/src/test/resources/bootstrap/standard.properties

```diff
@@ -45,6 +45,8 @@ staking.rewardHistory.numStoredPeriods=365
 # Dynamic properties
 accounts.maxNumber=5_000_000
 autoCreation.enabled=true
+lazyCreation.enabled=true
```

Comment:
Please also set it enabled in https://github.com/hashgraph/hedera-services/blob/master/hedera-node/data/config/bootstrap.properties
This should allow local-node testing sooner

Reply:
Just to confirm -> we should enable only `lazyCreation.enabled` and **not** `cryptoCreateWithAlias.enabled`?

Reply:
For local-node you can enable both so we are not waiting on testing.
For production environments both should be false in 0.31

Reply:
Got it --- enabled them now in [data/config/bootstrap.properties](https://github.com/hashgraph/hedera-services/pull/4044/files#diff-5c3047b01377706469e19cf373335ea7b653d31ccf6a33d585a2869ef996ba9eR4-R5)